src/util/CodeMirror.ts

```diff
@@ -38,6 +44,14 @@ export interface EditorMode {
     children?: EditorMode[];
 }
 
+export const DEFAULT_KEYMAP: string = 'vim';
+
+export const KEYMAPS: string[] = [
```

Comment:
if you know what the expected keymaps are you can do something in typescript like `export type Keymap = 'vim' | 'emacs' | 'sublime'` and then just use that type everywhere instead of `string`